auth/views.py

```diff
@@ -4,17 +4,30 @@
 
 from utils import prepare, send
 
+from braintree.environment import Environment
+
 
 def bango(request):
     raise NotImplementedError
 
 
 def braintree(request):
+    """
+    Pass the request through to Braintree. There are two jobs to do:
+    1) Add in the Braintree auth into the HTTP headers
+    2) Ensure that requests will check the correct Braintree crt.
+    """
     new_request = prepare(request)
+
+    # Add in the correct Braintree Authorization.
     new_request['headers']['Authorization'] = b"Basic " + encodebytes(
         settings.BRAINTREE_PUBLIC_KEY.encode('ascii') + b":" +
         settings.BRAINTREE_PRIVATE_KEY.encode('ascii')).strip()
-    return send('braintree', new_request)
+    # Taken from http://bit.ly/1cBESdC and ensures that the
+    # crt is passed through to the requests verify parameter.
+    new_request['verify'] = (
+        Environment.braintree_root() + '/ssl/api_braintreegateway_com.ca.crt')
```

Comment:
That would mean knowing the Environment in the proxy that is being requested. Either by configuration (another thing to configure) or by passing in the Environment by URL or HTTP Header from the server. I was trying to avoid the complexity, it might break in the future but I'd be against adding in complexity we don't need.